style.css

```diff
@@ -390,8 +399,117 @@ h4 {
   color: #42526e;
 }
 
+#empty-footer {
+  display: none;
+}
+
 footer {
   display: flex;
   flex-direction: column;
   align-items: center;
 }
+
+@media only screen and (min-width: 768px) {
+  .navbar {
+    display: block;
+    padding: 26px;
+    color: #344563;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  h2 {
+    display: none;
+  }
+
+  .header {
+    display: none;
+  }
+
+  a.arrow2 {
+    display: none;
+  }
+
+  a.arrow {
+    border: 1px solid #dfe1e6;
+    border-radius: 50%;
+    width: 40px;
+    height: 40px;
+    margin-top: 40px;
+    padding-bottom: 20px;
+    padding-right: 24px;
+    margin-left: 48%;
+    display: flex;
+  }
+
+  .landing-page {
+    background-image: url("./images/header-background.svg");
+    width: 100%;
+    text-align: center;
+  }
+
+  .proj-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 24px 32px;
+    margin-top: 52px;
+  }
+
+  .about-me {
+    max-width: 100%;
+    background-image: url("./images/illustration-about-me-desktop.png");
+    background-repeat: no-repeat;
+  }
+
+  .bg-img1,
+  .bg-img2 {
+    display: none;
+  }
+
+  #about-text p {
+    max-width: 920px;
+    padding-left: 10px;
+  }
+
+  .skills-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 24px 32px;
+    padding: 68px 0 0 0;
+  }
+
+  .contact-section {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    margin-bottom: 198px;
+  }
+
+  .btn-container {
+    display: none;
+  }
+
+  #btn {
+    display: block;
+    margin: 0;
+  }
+
+  .contact-section .contact-bg {
+    display: none;
+  }
+
+  .example-mail {
+    display: none;
+  }
+
+  #empty-footer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    gap: 20%;
+  }
```

Comment:
- [ ] Please decrease the space between the contact form and the footer, 

Your design 👇 
![Screen Shot 2022-06-09 at 23 08 40](https://user-images.githubusercontent.com/75126481/172945344-590c3b7e-68e3-4ff1-882c-aa7f984ba37c.png)

The original design 👇 

![Screen Shot 2022-06-09 at 23 09 54](https://user-images.githubusercontent.com/75126481/172945403-6115176f-e3ed-4402-bc68-914f169c280e.png)

Reply:
Done